nssdb/keys.go

```diff
@@ -0,0 +1,208 @@
+package nssdb
+
+import (
+	"context"
+	"crypto/ecdh"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"encoding/asn1"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"golang.org/x/crypto/cryptobyte"
+)
+
+// The only supported params for now
+var ecParams []byte
+
+func init() {
+	ecParams, _ = hex.DecodeString("06082a8648ce3d030107")
```

Comment:
That looks like an ASN.1 encoded OID. Can you add some information about this?

Reply:
https://github.com/smallstep/crypto/commit/453e3e724561006fd033ee8ae7e749eb92f7f156